docs/source-pytorch/conf.py

```diff
@@ -643,6 +643,7 @@ def package_list_from_file(file):
     r"starter/installation.html$",
     r"^../common/trainer.html#trainer-flags$",
     "https://deepgenerativemodels.github.io/assets/slides/cs236_lecture11.pdf",
+    "https://developer.habana.ai", # returns 403 error but redirects to intel.com documentation
```

Comment:
[some habana.ai  sub domains route to intel](https://habana.ai/press/intel-acquires-artificial-intelligence-chipmaker-habana-labs/). For some reason this just started causing 403 errors.

Reply:
https://developer.habana.ai redirects to https://www.intel.com/content/www/us/en/developer/platform/gaudi/overview.html